resources/dex/templates/dex-virtual-service.yaml

```
@@ -10,7 +10,7 @@ spec:
   hosts:
   - "dex.{{ .Values.global.ingress.domainName }}"
   gateways:
-  - {{ .Values.global.istio.gateway.namespace }}/{{ .Values.global.istio.gateway.name }}
+  - {{ .Values.global.istio.gateway.name }}.{{ .Values.global.istio.gateway.namespace }}
```

```suggestion
  - {{ .Values.global.istio.gateway.namespace }}/{{ .Values.global.istio.gateway.name }}
```

I don't agree here, according to https://istio.io/latest/docs/reference/config/networking/virtual-service/ it is correct format, please give a reasoning why you think your notation is more correct